### PR TITLE
add art-template support

### PR DESCRIPTION
**Pull request by farzer**
#116

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1681,3 +1681,107 @@ test('fastify.view with handlebars engine and callback in production mode', t =>
     })
   })
 })
+
+test('Art-template Test', t => {
+  const globalArt = require('art-template')
+  const templatePath = path.join(__dirname, 'templates', 'index.art')
+  globalArt.defaults.root = './temlates'
```
Comment (StarpTech) [inline test.js]: typo

Comment (farzer) [reply test.js]: can not be a test group? done

File: test.js [hunk 2]
```diff
@@ -1681,3 +1681,119 @@ test('fastify.view with handlebars engine and callback in production mode', t =>
...
+    sget({
+      method: 'GET',
+      url: 'http://127.0.0.1:10086/'
+    }, (err, response, body) => {
+      t.error(err)
+      t.strictEqual(response.statusCode, 200)
+      t.strictEqual(response.headers['content-length'], '' + body.length)
+      t.strictEqual(response.headers['content-type'], 'text/html; charset=utf-8')
+
+      art.defaults.root = './temlates'
```
Comment (Eomm) [inline test.js]: There are still the typo for all the `art.defaults.root`
Since the test are passing, is this line needed?

Comment (huaentandd01) [reply test.js]: fix

Comment (farzer) [reply test.js]: fix

**Pull request merged**